deploy/009-deploy-vtokens.ts

```diff
@@ -29,6 +29,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     hre,
   );
 
+  console.log("Is Time based", isTimeBased);
+
```

Comment:
@GitGuru7 maybe this console.log isn't intentional

Reply:
Yes, it's intentional to prevent any unintentional mistakes.